BoxContentSDK/BoxContentSDK/Requests/BOXFileCommentsRequest.h

```diff
@@ -10,6 +10,7 @@
 @property (nonatomic, readwrite, assign) BOOL requestAllItemFields;
 @property (nonatomic, readwrite, strong) NSURL *sharedLinkURL;
 @property (nonatomic, readwrite, strong) NSString *sharedLinkPassword; // Only required if the shared link is password-protected
+@property (nonatomic, readonly, strong) NSString *fileID;
```

Comment:
Good call, however let's remain consistent with the rest of the project right now, which is still adhering to this pattern. We should probably do a round of clean house afterwards.